src/controllers/Auth/loginEmployees.ts

```diff
@@ -0,0 +1,21 @@
+import jwt from 'jsonwebtoken';
+import { empleados } from '../../fakedata/empleados.json';
+import { constants } from '../../config/constants'
+import { Request, Response } from 'express';
+
+export default(req: Request, res: Response) => {
+    const credentials = req.body;
+
+    const match = empleados.find(employee => {
+        if(employee.id === credentials.id && employee.nombre === credentials.nombre){
+            return employee;
+        }
+    });
+    if(!match){
+        return res.sendStatus(404);
+    }
```

Comment:
**antes y después** de un bloque IF, MAP, FOR, FILTER, REDUCE, etc, deja espacios